parsec/data_dist/matrix/matrix.c

```diff
@@ -337,7 +338,7 @@ int parsec_tiled_matrix_data_read(parsec_tiled_matrix_t *tdesc, char *filename)
                     buf = parsec_data_get_ptr(data, 0);
                     for (k=0; k < tdesc->nb; k++) {
                         ret = fread(buf, eltsize, tdesc->mb, tmpf );
-                        if ( ret !=  tdesc->mb ) {
+                        if ( ret !=  (size_t)tdesc->mb ) {
```

Comment:
If you change the type of `ret` you should also change the format specifier below to `%zu`

Reply:
Fixed

Reply:
Should `tdesc->mb` also be changed to `size_t`?

Reply:
`mb` should not be that large, right?